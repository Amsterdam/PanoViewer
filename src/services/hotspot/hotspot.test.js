import createHotspot from './hotspot';

describe('hotspot', () => {
  it('should create the hotspot', () => {
    const hotspot = createHotspot(90, 50);
    expect(hotspot).toMatchSnapshot();
  });
});
