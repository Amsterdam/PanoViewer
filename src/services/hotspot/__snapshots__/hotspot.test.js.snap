// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hotspot should create the hotspot 1`] = `
<div
  class="c-panoviewer-hotspot"
>
  <button
    class="c-panoviewer-hotspot__content"
    style="transform: rotateX(90deg) scale(0.2);"
  />
</div>
`;
